contracts/FlowIDTableStaking.cdc

```diff
@@ -982,10 +986,26 @@ pub contract FlowIDTableStaking {
         return FlowIDTableStaking.nodes.keys
     }
 
+    /// Gets the total amount of tokens that have been staked and
+    /// committed for a node for the next epoch
+    pub fun getNodeCommittedBalanceWithoutDelegators(_ nodeID: String): UFix64 {
+        let nodeRecord = self.borrowNodeRecord(nodeID)
+
+        if (nodeRecord.tokensCommitted.balance + nodeRecord.tokensStaked.balance) < nodeRecord.tokensRequestedToUnstake {
+            return 0.0
+        } else {
+            var sum: UFix64 = 0.0
+
+            sum = nodeRecord.tokensCommitted.balance + nodeRecord.tokensStaked.balance - nodeRecord.tokensRequestedToUnstake
+
+            return sum
```

Comment:
Could this be a one-liner?

Reply:
yes it could be simpler. I'll change it. Good suggestion!